src/api/FetchApi.js

```diff
@@ -0,0 +1,26 @@
+const fetch = require("node-fetch");
+
+module.exports = class FetchApi {
+    constructor(baseUrl) {
+        this.baseUrl = baseUrl ? baseUrl : "https://api.github.com/";
+    }
+
+    getBaseUrl() {
+        return this.baseUrl;
+    }
+
+    /**
+     * Make a request to GitHub public API
```

Comment:
Maybe "Make a request to GitHub public API" needs to be change to "Make a request to public API".

Reply:
👍